src/main/java/reactor/ipc/netty/http/HttpOperations.java

```diff
@@ -187,4 +244,24 @@ protected final boolean markHeadersAsSent() {
 			AtomicIntegerFieldUpdater.newUpdater(HttpOperations.class,
 					"statusAndHeadersSent");
 
+	final static ChannelInboundHandler HTTP_EXTRACTOR = NettyPipeline.inboundHandler(
+			(ctx, msg) -> {
+				if (msg instanceof ByteBufHolder) {
+					ByteBuf bb = ((ByteBufHolder) msg).content();
```

Comment:
(continuing) the 2 lines above would have been the user-provided lambda, with `else return msg`